src/js/rx/components/SubmitRefill.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import classNames from 'classnames';
+
+class SubmitRefill extends React.Component {
+  render() {
+    // Using !== so that it will default to usa-button-outline even if
+    // no `mode` property is defined.
+    const classList = classNames({
+      'usa-button-outline': this.props.mode !== 'compact',
+      'usa-button-unstyled': this.props.mode === 'compact',
+    });
+
+    return (
+      <form className="rx-refill-form" onSubmit={this.props.onSubmit}>
+        <input type="hidden" name="refillId" value={this.props.refillId}/>
+        <button
+            className={`${classList} ${this.props.cssClass}`}
```

Comment:
You can actually pass `this.props.cssClass` as an argument to `classNames`, I think.